### PR TITLE
Update the debian stack to be able to package matomo 4.x for Buster or Bullseye

### PR DESCRIPTION
**Pull request by e-gaulue**
Review all Debian stack.

Only left lintian warnings as our script is using kinds of `chmod -R` on /usr/share/matomo/tmp. Debian discourage those `-R` on `chmod`. There shouldn't be any risk, but maybe a discussion is needed.

Documentation in README.md is still valid.

Doing this you get a deb package, ou can use.

### CODE REVIEW

**Comment by e-gaulue** [comment]
I will work tonight on a changelog text, summing up all has been done.

**Comment by e-gaulue** [comment]
Next changelog could be:
```
 * deal with new files in 4.x (focus on Buster and Bulleyes due to min php version) 
 * fix lintian overrides (add/delete libraries/licenses/fonts...)
 * change Makefile to reflect lintian request (import_logs.py use python3, fix permissions)
 * postinstall now execute: /usr/share/matomo/console core:create-security-files
 * review /etc/matomo/apache.conf to reflect the previous line
 * fix debian packages dependencies
 * fix package compatibility level to 13
 * update copyright
 * update Debian.md
```

File: debian/matomo.postinst [hunk 1]
```diff
@@ -102,6 +108,10 @@ else
 	echo "  * To finish Matomo's upgrade, please run: /usr/bin/php /usr/share/matomo/console core:update"
 fi
 
+if [ -e "/usr/share/matomo/config/config.ini.php" ]; then
+	echo "  * If upgradingr, visit index.php?module=Installation&action=systemCheckPage to check and manage your plugins."
```
Comment (Findus23) [inline debian/matomo.postinst:112]:
```suggestion
	echo "  * If upgrading, visit index.php?module=Installation&action=systemCheckPage to check and manage your plugins."
```

**Review by Findus23 — COMMENTED** [review]
I'm not the biggest expert on Debian Packaging, but all changes seem reasonable to me.

File: debian/matomo.postinst (hunk 1 above)
Comment (e-gaulue) [reply debian/matomo.postinst:112]: Of course, that's a typo.

**Pull request closed without merging**